src/templates/static/get_started/binary_options.jsx

```diff
@@ -236,12 +236,12 @@ const BinaryOptions = () => {
                             <ListStrong header={it.L('Even/Odd')}        text={it.L('Predict that the last digit is an even number or odd number after the last tick')} />
                             <ListStrong header={it.L('Over/Under')}      text={it.L('Predict that the last digit is higher or lower')} />
                         </SectionLarge>
-                        <SectionLarge dataAnchor='reset-callput' header={it.L('Reset Call/Put')} image='types-of-trades/reset' text={it.L('There are two types of reset trades:')}>
+                        <SectionLarge dataAnchor='reset-callput' header={it.L('Reset Call/Reset Put')} image='types-of-trades/reset' text={it.L('There are two types of reset trades:')}>
                             <ListStrong header={it.L('Reset-Call')} text={it.L('Predict that the market will end up higher than either the current level or the level at a predetermined time (reset time)')} />
```

Comment:
should this be changed to Reset Call (without the dash)?

Reply:
Yep, Dervin specifically told us to use that (without dash)

Reply:
https://binarygroup.mojohelpdesk.com/ma/#/tickets/21983121

Reply:
@ahkuma line 240 needs to be changed to **Reset Call** and line 241 to **Reset Put**😄